_rules/additional-content-triggered-on-hover-is-hoverable-ep1s13.md

```diff
@@ -0,0 +1,161 @@
+---
+id: ep1s13
+name: Additional content triggered on hover is hoverable
+rule_type: atomic
+description: |
+  This rule checks that any additional content that was triggered by hovering an element remains available to the user while the mouse pointer is over the element or the additional content
+accessibility_requirements:
+  wcag21:1.4.13: # Content on Hover or Focus (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Mouse pointer
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any element that when [hovered][] causes [visible changes in content][] and for the [area affected by the changes][area affected by visible changes] all the following are true:
+
+- the area is [adjacent][adjacent areas] or [overlaps][overlapping areas] the [bounding box][] of the target element; and
+- the area is not [contained][] inside the [bounding box][] of the target element.
+
+## Expectation
+
+There are no [visible changes in content][] in the [area affected by the changes][area affected by visible changes] while the target element is [hovered][] or the mouse pointer remains inside that area.
+
+## Assumptions
+
+- The user does not dismiss the tooltip causing the changes in content by pressing a key on the keyboard. In this instance the rule will fail while [success criterion 1.4.13: Content on Hover or Focus][sc1.4.13] might be satisfied.
+- The content displayed on the rectangle with the changes in content does not become irrelevant. In this instance the rule will fail while [success criterion 1.4.13: Content on Hover or Focus][sc1.4.13] might be satisfied.
+- The additional content displayed on hover does not include animations. An animation would cause visible changes to the content of the page and therefore fail this rule. However, if the animation is displayed only while the mouse pointer hovers above the target element or the rectangle with the changes, [success criterion 1.4.13: Content on Hover or Focus][sc1.4.13] might be satisfied.
```

Comment:
I don't think this is a reasonable assumption. Animations, especially fade-in and face-out animations are common place on the web.

Reply:
Fade-out animations are irrelevant to this rule. Would a delay before checking the expectation (to give time for the fade-in animation to complete) be acceptable to you?